datatables/nodejs/index.js

```diff
@@ -134,9 +134,12 @@ function convertToType(val, header){
 }
 
 // Authenticate
-const clientId = process.env.GENESYS_CLOUD_CLIENT_ID;
-const clientSecret = process.env.GENESYS_CLOUD_CLIENT_SECRET;
-client.loginClientCredentialsGrant(clientId, clientSecret)
+const GENESYS_CLOUD_CLIENT_ID = process.env.GENESYS_CLOUD_CLIENT_ID;
+const GENESYS_CLOUD_CLIENT_SECRET = process.env.GENESYS_CLOUD_CLIENT_SECRET;
+const GENESYS_CLOUD_CLIENT_REGION = 'mypurecloud.com';
```

Comment:
Yeah an environment variable would be best to use here for consistency

Reply:
Hardcoded environment/region values updated to be read from env variables.